src/components/HockeyStatTracker/HockeyStatTrackerSecondary.java

```diff
@@ -89,16 +95,56 @@ public boolean equals(Object obj) {
 
     // Abstract kernel methods to be implemented by concrete subclass
 
+    /**
+     * Returns all players currently tracked.
+     *
+     * @return an iterable collection of player names
+     */
     protected abstract Iterable<String> getAllPlayers();
```

Comment:
These are all inherited I would assume, so they are not needed and neither are the docs.